setup.py

```diff
@@ -66,7 +64,8 @@
         "cloudpickle >= 1.2.0",
         "importlib_metadata >= 4.8.0; python_version < '3.10'",
         "gymnasium_notices >= 0.0.1",
-        "dataclasses == 0.8; python_version == '3.6'",
+        "brax-jumpy>=0.1.0",
```

Comment:
We have updated the project name to jax-jumpy, https://pypi.org/project/jax-jumpy/

Reply:
fixed :+1: